### DISCUSSION TITLE
Which public clouds and regions are supported?

### DISCUSSION THREAD

**Discussion by emoido** [ChistaDATA/Cloud#8] [❓FAQ] [answered]
Which public clouds and regions are supported?

**Comment by TheraniAA** [comment, marked as answer]
ChistaDATA DBaaS currently supports Amazon Web Services (AWS) and we plan to support Google Cloud Platform (GCP), and Microsoft Azure public cloud in the future. The specific regions supported may vary depending on the cloud provider and the ChistaDATA DBaaS service plan selected by the user.
